src/translation/en/en.json

```diff
@@ -114,5 +114,19 @@
   "You will see the following after joining the team": "You will see the following after joining the team:",
   "To leave the team": "To leave the team, click the “Leave Team” button or ask someone from your teammates to exclude you from the team.",
   "Data is unsaved": "Data is unsaved",
-  "Are sure want to leave page without saving data?": "Are sure want to leave page without saving data?"
```

Comment:
Return!

Reply:
"Are **you** sure want to leave page without saving data?"